src/Common/src/Interop/Windows/SspiCli/SecuritySafeHandles.cs

```diff
@@ -644,7 +644,7 @@ internal abstract partial class SafeDeleteContext : SafeHandle
             }
 
             Interop.SspiCli.SecBufferDesc inSecurityBufferDescriptor = new Interop.SspiCli.SecBufferDesc(inSecBuffers.Length);
-            Interop.SspiCli.SecBufferDesc outSecurityBufferDescriptor = new Interop.SspiCli.SecBufferDesc(1);
+            Interop.SspiCli.SecBufferDesc outSecurityBufferDescriptor = new Interop.SspiCli.SecBufferDesc(2);
```

Comment:
Not clear what the 2 is doing here.

```suggestion
            Interop.SspiCli.SecBufferDesc outSecurityBufferDescriptor = new Interop.SspiCli.SecBufferDesc(count: 2);
```

Reply:
This allocates fixed size array with pointer to individual buffers. In past, we would always use just one e.g. frame to be sent out. Now we also use buffer for alerts. That would be initially NULL and it would be allocated by schannel as needed. 
I made change as you suggested but I'm not sure if that helps to explain what is going on.